src/lib/core/style/_input-common.scss

```diff
@@ -0,0 +1,36 @@
+@import './variables';
+
```

Comment:
Add top-level description comment that mentions these are styles used by input-style components, including md-input-container and md-select